app/controllers/api/v1/merchants/revenue_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V1::Merchants::RevenueController < ApplicationController
+
+  def show
```

Comment:
Ah, ok. So searching by created_at and updated_at will not get picked up in this first section. Cool